controllers/contact-controller.js

```diff
@@ -171,7 +204,8 @@ function isValidContact(contact) {
     !isNullOrEmptyObject(contact.rawJson.callType) &&
     !isNullOrEmptyObject(contact.rawJson.childInformation) &&
     !isNullOrEmptyObject(contact.rawJson.callerInformation) &&
-    !isNullOrEmptyObject(contact.rawJson.caseInformation)
+    !isNullOrEmptyObject(contact.rawJson.caseInformation) &&
+    nonDataOrSummary(contact)
```

Comment:
I think we should delete the offending rows of `Contacts` in prod and staging, and don't need to add this.  We have been iterating on a schema with throwaway data, and there's no need to add special case code that might live forever in our codebase to handle our old scratch work.  The time around, the simplicity of our codebase is more important than the data (this is never true with real customer data, of course).  So we can remove this and I'll follow up with you privately about deletion.